crates/control/src/motion/stand_up_back.rs

```diff
@@ -59,13 +62,16 @@ impl StandUpBack {
     }
 
     pub fn cycle(&mut self, context: CycleContext) -> Result<MainOutputs> {
+        let mut stand_up_back_remaining_duration = Duration::ZERO;
         if let MotionType::StandUpBack = context.motion_selection.current_motion {
             self.advance_interpolator(context);
+            stand_up_back_remaining_duration = self.interpolator.remaining_estimated_duration();
         } else {
             self.interpolator.reset();
         };
         Ok(MainOutputs {
             stand_up_back_positions: self.interpolator.value().into(),
+            stand_up_back_remaining_duration: stand_up_back_remaining_duration.into(),
```

Comment:
please also put something like estimated into the name, since the time could deviate quite a bit when conditions are being used

Reply:
done